docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       - db
 
   db:
-    build: https://github.com/genome-nexus/genome-nexus-importer.git
+    build: https://github.com/leexgh/genome-nexus-importer.git#18eee93d7ad38c2cc99319008e4cac1cc2dbca6c
```

Comment:
will update this after we merge the https://github.com/genome-nexus/genome-nexus-importer/pull/30